### DISCUSSION TITLE
Is there a chunk generator available that represents normal terrain generation?

### DISCUSSION THREAD

**Discussion by andromdaa** [Minestom/Minestom#786] [Help]
Just the title, looking for a chunk generator which allows for default terrain generation.

**Comment by Flamgop** [comment]
Currently there are no chunk generators that look like default terrain generation. I believe multiple people (including me) are working on it though.

**Reply by andromdaa** [reply]
Alright thank you. If it isn't too much trouble, I would love to learn if you could point me in the correct direction or simply suggest material to cover?

**Reply by BomBardyGamer** [reply]
I've studied enough of vanilla's world generation to offer you some useful words of advice: Don't even bother getting started. It'll take you months and months of reading and researching and documentation and other stuff to try and understand it. There is currently a kinda solution in the works, where Minestom uses vanilla to generate worlds, though I'm not sure how well that actually works. I think there was a showcase of it somewhere. If you look back in the showcase channel on Discord, eventually you'll find it.
